Sources/TuistKit/Loader/Graph.swift

```diff
@@ -46,12 +47,18 @@ protocol Graphing: AnyObject {
     func linkableDependencies(path: AbsolutePath, name: String) throws -> [DependencyReference]
     func librariesPublicHeadersFolders(path: AbsolutePath, name: String) -> [AbsolutePath]
     func embeddableFrameworks(path: AbsolutePath, name: String, system: Systeming) throws -> [DependencyReference]
-    func dependencies(path: AbsolutePath, name: String) -> Set<GraphNode>
-    func dependencies(path: AbsolutePath) -> Set<GraphNode>
     func targetDependencies(path: AbsolutePath, name: String) -> [String]
+    func staticLibraryDependencies(path: AbsolutePath, name: String) -> [DependencyReference]
+    
+    // MARK:- DFS
```

Comment:
What does DFS stand for? Would you mind expanding it?

Reply:
Depth First Search, I'll add some header documentation to what it is and why we need it.